### DISCUSSION TITLE
Copy of a product

### DISCUSSION THREAD

**Discussion by Rodien** [openstore-ecommerce/OpenStore#246] [Q&A] [answered]
Quick question...

Is there a reason why when you copy a product categories and properties are not copied? 

@leedavi @DNNMonster 

Thanks

**Comment by leedavi** [comment, marked as answer]
Well, this reason is the product xref are separate records.

But the answer to your question…… it wasn’t required by the client that asked for the copy to be added.

We could added it to  the issues?  But I doubt it'll get done unless there is someone willing to pay for it.

**Reply by Rodien** [reply]
I just sent you a mail about this.
